### DISCUSSION TITLE
my git page dont show the chosen theme and posts don't work

### DISCUSSION THREAD

**Discussion by GuntherVD** [community/community#21784] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>Im trying to get the git page to work on my account.</p>
<p>I have 2 isues i cant figure</p>
<ul>
<li>
<p>The chosen theme doesn’t show</p>
</li>
<li>
<p>The post’s don’t render.</p>
</li>
</ul>
<p>I have folowed the guide on git to set it up but im realy lost now.</p>
<p>The repo is here:</p>
<p><a href="https://github.com/GuntherVD/GuntherVD.github.io" rel="nofollow noopener">https://github.com/GuntherVD/GuntherVD.github.io</a></p>
<p>The page is here:</p>
<p><a href="https://gunthervd.github.io/" rel="nofollow noopener">https://gunthervd.github.io/</a></p>
<p>On the repo i dont get rendering errors, allso in my console i don’t see problems.</p>
<p>Anyone got a idea whet im doing wrong?</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/gunthervd">@gunthervd</a>,</p>
<p>It sounds like your question is actually about a GitHub Pags site, so I’ve moved this post to the Pages board. This means you’ll get better engagement on your post from other Community Forum members, and it keeps our Community organized so users can more easily find information.</p>
<p>Cheers!</p>

**Comment by GuntherVD** [comment]
<p>Sorry my bad for writing in the wrong forum.</p>
<p>I have some aditional info:</p>
<p>I have installed jekyll localy end when i run the server i get the following message:</p>
<p>Build Warning: Layout ‘feed’ requested in blog/atom.xml does not exist.</p>
<p>All the links in the atom file do exist. im totaly lost.</p>
<p>Allso localy the posts don’t work the style either.</p>

**Comment by GuntherVD** [comment]
<p>I localy generated the theme with ruby using:</p>
<h1>git clone <a href="https://github.com/pages-themes/hacker" rel="nofollow noopener">https://github.com/pages-themes/hacker</a>
</h1>
<p>then published using:</p>
<h1>git commit -m ‘First commit’</h1>
<h1>git push origin master</h1>
<p>I will start from there to build my page.</p>

**Comment by mpboom** [comment, marked as answer]
<p>Follow this guide: <a href="https://guides.github.com/features/pages/" rel="nofollow noopener">https://guides.github.com/features/pages/</a>.</p>

**Comment by ToniWonKanobi** [comment]
<p>I have been having the same problem.</p>
<p>I followed <a href="http://jmcglone.com/guides/github-pages/" rel="nofollow noopener">this</a> guide, which suggests the following:</p>
<p>&gt; Inside your blog/ directory create a file and name it atom.xml. Add this to file:</p>
<p>&gt; ```</p>
<p>&gt; —</p>
<p>&gt; layout: feed</p>
<p>&gt; —</p>
<p>&gt; &lt;?xml version=“1.0” encoding=“utf-8”?&gt;</p>
<p>&gt; &lt;feed xmlns=“<a href="http://www.w3.org/2005/Atom" rel="nofollow noopener">http://www.w3.org/2005/Atom</a>”&gt;</p>
<p>&gt;</p>
<p>&gt;  &lt;title&gt;Hank Quinlan’s Blog&lt;/title&gt;</p>
<p>&gt;  &lt;link href=“<a href="http://hankquinlan.github.io/blog/atom.xml" rel="nofollow noopener">http://hankquinlan.github.io/blog/atom.xml</a>” rel=“self”/&gt;</p>
<p>&gt;  &lt;link href="<a href="http://hankquinlan.github.io/blog%22/" rel="nofollow noopener">http://hankquinlan.github.io/blog"/</a>&gt;</p>
<p>&gt;  &lt;updated&gt;{{ site.time | date_to_xmlschema }}&lt;/updated&gt;</p>
<p>&gt;  &lt;id&gt;<a href="http://hankquinlan.github.io/blog%3C/id" rel="nofollow noopener">http://hankquinlan.github.io/blog&lt;/id</a>&gt;</p>
<p>&gt;  &lt;author&gt;</p>
<p>&gt;    &lt;name&gt;Hank Quinlan&lt;/name&gt;</p>
<p>&gt;    &lt;email&gt;<a href="mailto:hankquinlanhub@gmail.com">hankquinlanhub@gmail.com</a>&lt;/email&gt;</p>
<p>&gt;  &lt;/author&gt;</p>
<p>&gt;</p>
<p>&gt;  {% for post in site.posts %}</p>
<p>&gt;    &lt;entry&gt;</p>
<p>&gt;      &lt;title&gt;{{ post.title }}&lt;/title&gt;</p>
<p>&gt;      &lt;link href=“<a href="http://hankquinlan.github.io" rel="nofollow noopener">http://hankquinlan.github.io</a>{{ post.url }}”/&gt;</p>
<p>&gt;      &lt;updated&gt;{{ post.date | date_to_xmlschema }}&lt;/updated&gt;</p>
<p>&gt;      &lt;id&gt;<a href="http://hankquinlan.github.io" rel="nofollow noopener">http://hankquinlan.github.io</a>{{ post.id }}&lt;/id&gt;</p>
<p>&gt;      &lt;content type=“html”&gt;{{ post.content | xml_escape }}&lt;/content&gt;</p>
<p>&gt;    &lt;/entry&gt;</p>
<p>&gt;  {% endfor %}</p>
<p>&gt;</p>
<p>&gt; &lt;/feed&gt;</p>
<p>&gt; ```</p>
<p>And I’ve been getting the same error as the OP:</p>
<pre><code class="lang-auto">
Build Warning: Layout 'feed' requested in blog/atom.xml does not exist.

</code></pre>
<p>when I serve the site locally with <code>jekyll serve</code></p>
<p>Thoughts?</p>

**Comment by ghost** [comment]
<p>That error means Jekyll is looking for a layout named “feed.html” in the _layouts directory, and not finding it. (That’s what the “layout: feed” at the top of your source file means: Use the layout named “feed” for this file.)</p>
<p>This appears to be an error in the guide you’re following. He doesn’t discuss creating that layout, and it’s not in the downloadable ZIP file with sources.</p>

**Comment by matteoaiello** [comment]
<p>Hank isn’t as clever as he thinks he is. He forgets to give you a “feed.html” format file in the _layouts directory. You are basically calling a file format that does not exist. You have default and you have post, but you have no feed. </p>
